script/tasks/updateFacetConfig.sh

```
@@ -77,10 +77,15 @@ updateFacetConfig() {
       SKIP_SIMULATION_FLAG=$(getSkipSimulationFlag)
-      
+
+      local TEMPO_PROFILE_PREFIX
+      TEMPO_PROFILE_PREFIX=$(getTempoForgeProfilePrefix "$NETWORK")
+      local LEGACY_CLI_FLAG
+      LEGACY_CLI_FLAG=$(getForgeLegacyCliFlag "$NETWORK")
+
       # Execute, parse, and check return code
       if ! executeAndParse \
-        "NETWORK=$NETWORK FILE_SUFFIX=$FILE_SUFFIX USE_DEF_DIAMOND=$USE_MUTABLE_DIAMOND PRIVATE_KEY=$(getPrivateKey \"$NETWORK\" \"$ENVIRONMENT\") forge script \"$SCRIPT_PATH\" --fork-url \"$NETWORK\" --json --broadcast --legacy $SKIP_SIMULATION_FLAG --gas-estimate-multiplier \"$GAS_ESTIMATE_MULTIPLIER\"" \
+        "${TEMPO_PROFILE_PREFIX}NETWORK=$NETWORK FILE_SUFFIX=$FILE_SUFFIX USE_DEF_DIAMOND=$USE_MUTABLE_DIAMOND PRIVATE_KEY=$(getPrivateKey \"$NETWORK\" \"$ENVIRONMENT\") forge script \"$SCRIPT_PATH\" --fork-url \"$NETWORK\" --json --broadcast ${LEGACY_CLI_FLAG}$SKIP_SIMULATION_FLAG --gas-estimate-multiplier \"$GAS_ESTIMATE_MULTIPLIER\"" \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)